mrp_project_analytic/models/mrp_production.py

```diff
@@ -40,6 +40,14 @@ def mark_analytic_mrp_from_child(self):
         )
         aml_ids = raw_material_moves.sudo().mapped('account_move_ids.line_ids')
         
+        for move in self.move_raw_ids.filtered(lambda x: x.state == "done"):
+            mr_qty = sum(self.search([('project_id', '=', self.project_id.id), ('state', '=', 'done')]).move_raw_ids.filtered(lambda x: x.product_id == move.product_id.id).mapped('quantity_done'))
```

Comment:
`self` is could be `multi`, because `button_mark_done` is `multi`, so `self.project_id.id` could be raise an error. Such call should be replaced by "get the project from the production of current move" instead.